.circleci/config.yml

```diff
@@ -0,0 +1,128 @@
+version: 2.1
+orbs:
+  node: circleci/node@5.0.2
+jobs:
+  build:
+    executor: node/default
+    steps:
+      - checkout
+      - node/install-packages:
+          pkg-manager: npm
+      - run:
+          command: sudo npm install -g ts-node
+          name: Install ts-node
+      - run:
+          command: npm ci
+          name: Install NPM dependencies for fmt/lint
+      - run:
+          command: ts-node -v
+          name: Check ts-node version to make sure it installed
+      - persist_to_workspace:
+          root: .
+          paths:
+            - '*'
+
+  test:
+    executor: node/default
+    steps:
+      - attach_workspace:
+          at: .
+      - checkout
+      - run:
+          command: npm run test
+          name: run tests
+
+  format:
+    executor: node/default
+    steps:
+      - attach_workspace:
+          at: .
+      - checkout
+      - run:
+          name: format
+          command: npm run format
```

Comment:
I think we need to use a `format` that shows errors, like this: https://github.com/bloXroute-Labs/solana-trader-api/blob/4c1e334fbdabc0f865fcd8449a98c7edf064fc56/.circleci/config.yml#L127

Can you test this by adding a format error, checking that the format error shows up in CircleCI (red test), and then removing the format error?